File: data/imported/knowledge/slides/cpp/merged/snippets/snippet_009.cpp
TEST(ConnectorIntegrationTestNoFixture, AddStraightConnector) {
    Presentation pres;
    auto& slide = pres.slides()[0];
    auto& conn = slide.shapes().add_connector(
        ShapeType::STRAIGHT_CONNECTOR1, 100, 100, 300, 200);
    EXPECT_EQ(conn.shape_type(), ShapeType::STRAIGHT_CONNECTOR1);
}

File: data/imported/knowledge/slides/java/merged/snippets/snippet_009.java
@Test
    void testClearComments() {
        try (Presentation pres = new Presentation()) {
            ICommentAuthor author = pres.getCommentAuthors().addAuthor("Alice", "A");
            ISlide slide = pres.getSlides().get(0);
            LocalDateTime now = LocalDateTime.now();
            author.getComments().addComment("C1", slide, new PointF(1, 1), now);
            author.getComments().addComment("C2", slide, new PointF(2, 2), now);
            author.getComments().clear();
            assertThat(author.getComments().size()).isEqualTo(0);
        }
    }